packages/wallet/hardhat-b.config.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
can we rename this file to hardhat2.config.js ..? minor distinction, but its the convention we follow in packages/0xsequence/hardhat.config.js and packages/0xsequence/hardhat2.config.js